### PR TITLE
Chore: Update metrics for AWS/MediaConnect

### LINKED ISSUE #74883: CloudWatch: Update metrics for AWS/MediaConnect
We need to update the [hardcoded metrics](https://github.com/grafana/grafana/blob/4b7b323061a803125db161219c5452d7e2f27e6f/pkg/tsdb/cloudwatch/constants/metrics.go#L360) for AWS/MediaConnect to include newer metrics ([aws docs](https://docs.aws.amazon.com/mediaconnect/latest/ug/monitor-with-cloudwatch-metrics-source-health.html))

### PR DESCRIPTION
**Pull request by Deepali1211**
**What is this feature?**

Added newer metrics for AWS/MediaConnect

**Why do we need this feature?**

To be updated

**Who is this feature for?**

grafana devs

**Which issue(s) does this PR fix?**:

Fixes #74883

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by iwysiu — COMMENTED** [review]
Hi @Deepali1211 ! Thanks for the contribution, it's a lot of metrics! Looking at the docs, it seems like we may still be missing the [maintenance metrics](https://docs.aws.amazon.com/mediaconnect/latest/ug/monitor-with-cloudwatch-metrics-flow-health.html#monitor-with-cloudwatch-metrics-flow-health-maintenance) Also, could you alphabetize the added metrics?

File: pkg/tsdb/cloudwatch/constants/metrics.go [hunk 1]
```diff
@@ -357,7 +357,7 @@ var NamespaceMetricsMap = map[string][]string{
...
...nceFailed", "NotRecoveredPackets", "OutputConnected", "OutputDisconnections", "OutputTotalBytes", "OutputDroppedPayloads", "OutputLatePayloads", "OutputTotalPayloads", "OverflowPackets", "PATError", "PCRAccuracyError", "PCRError", "PIDError", "PMTError", "PTSError", "PacketLossPercent", "RecoveredPackets", "RoundTripTime", "SourceARQRecovered", "SourceARQRequests", "SourceBitRate", "SourceCATError", "SourceCRCError", "SourceConnected", "SourceContinuityCounter", "SourceDisconnections", "SourceDroppedPackets", "SourceFECPackets", "SourceFECRecovered", "SourceMergeActive", "SourceSelected", "SourceMergeLatency", "SourceMergeStatusWarnMismatch", "SourceMergeStatusWarnSolo", "SourceNotRecoveredPackets", "SourceMissingPackets", "SourceOverflowPackets", "SourcePATError", "SourcePCRAccuracyError", "SourcePCRError", "SourcePIDError", "SourcePMTError", "SourcePTSError", "SourcePacketLossPercent", "SourceRecoveredPackets", "SourceRoundTripTime", "SourceTSByteError", "SourceTSSyncLoss", "SourceTotalPackets", "SourceTotalBytes", "SourceDroppedPayloads", "SourceLatePayloads", "SourceTotalPayloads", "SourceTransportError", "TSByteError", "TSSyncLoss", "TotalPackets", "TransportError", "Uptime"},
```
Comment (iwysiu) [inline pkg/tsdb/cloudwatch/constants/metrics.go:360]: Looks good, but just fixing some of the alphabetization
```suggestion
	"AWS/MediaConnect":         {"ARQRecovered", "ARQRequests", "BitRate", "CATError", "Connected", "ConnectedOutputs", "ContinuityCounter", "ConsecutiveDrops", "ConsecutiveNotRecovered", "ConnectionAttempts", "CRCError", "Disconnections", "DroppedPackets", "EgressBridgeBitRate", "EgressBridgeCATError", "EgressBridgeContinuityCounter", "EgressBridgeCRCError", "EgressBridgeDroppedPackets", "EgressBridgeFailoverSwitches", "EgressBridgeMergeActive", "EgressBridgeNotRecoveredPackets", "EgressBridgePacketLossPercent", "EgressBridgePATError", "EgressBridgePCRAccuracyError", "EgressBridgePCRError", "EgressBridgePIDError", "EgressBridgePMTError", "EgressBridgePTSError", "EgressBridgeRecoveredPackets", "EgressBridgeSourceBitRate", "EgressBridgeSourceCATError", "EgressBridgeSourceCRCError", "EgressBridgeSourceContinuityCounter", "EgressBridgeSourceDroppedPackets", "EgressBridgeSourceMergeActive", "EgressBridgeSourceMergeLatency", "EgressBridgeSourceNotRecoveredPackets", "EgressBridgeSourcePacketLossPercent", "EgressBridgeSourcePATError", "EgressBridgeSourcePCRAccuracyError", "EgressBridgeSourcePCRError", "EgressBridgeSourcePIDError", "EgressBridgeSourcePMTError", "EgressBridgeSourcePTSError", "EgressBridgeSourceRecoveredPackets", "EgressBridgeSourceTotalPackets", "EgressBridgeSourceTransportError", "EgressBridgeSourceTSByteError", "EgressBridgeSourceTSSyncLoss", "EgressBridgeTotalPackets", "EgressBridgeTransportError", "EgressBridgeTSByteError", "EgressBridgeTSSyncLoss",  "FailoverSwitches", "IngressBridgeBitRate", "IngressBridgeCATError", "IngressBridgeContinuityCounter", "IngressBridgeCRCError", "IngressBridgeDroppedPackets", "IngressBridgeFailoverSwitches", "IngressBridgeMergeActive", "IngressBridgeNotRecoveredPackets", "IngressBridgePacketLossPercent", "IngressBridgePATError", "IngressBridgePCRAccuracyError", "IngressBridgePCRError", "IngressBridgePIDError", "IngressBridgePMTError", "IngressBridgePTSError", "IngressBridgeRecoveredPackets", "IngressBridgeSourceARQRecovered", "IngressBridgeSourceARQRequests", "IngressBridgeSourceBitRate", "IngressBridgeSourceCATError", "IngressBridgeSourceCRCError", "IngressBridgeSourceContinuityCounter", "IngressBridgeSourceDroppedPackets", "IngressBridgeSourceFECPackets", "IngressBridgeSourceFECRecovered", "IngressBridgeSourceMergeActive", "IngressBridgeSourceMergeLatency", "IngressBridgeSourceNotRecoveredPackets", "IngressBridgeSourceOverflowPackets", "IngressBridgeSourcePATError", "IngressBridgeSourcePacketLossPercent", "IngressBridgeSourcePCRAccuracyError", "IngressBridgeSourcePCRError", "IngressBridgeSourcePIDError", "IngressBridgeSourcePMTError", "IngressBridgeSourcePTSError", "IngressBridgeSourceRecoveredPackets", "IngressBridgeSourceRoundTripTime","IngressBridgeSourceTotalPackets", "IngressBridgeSourceTransportError",  "IngressBridgeSourceTSByteError", "IngressBridgeSourceTSSyncLoss", "IngressBridgeTotalPackets", "IngressBridgeTransportError", "IngressBridgeTSByteError", "IngressBridgeTSSyncLoss", "Jitter", "Latency", "MaintenanceCanceled", "MaintenanceRescheduled", "MaintenanceScheduled", "MaintenanceStarted", "MaintenanceSucceeded", "MaintenanceFailed", "MergeActive", "MergeLatency", "NotRecoveredPackets", "OutputConnected", "OutputDisconnections", "OutputDroppedPayloads", "OutputLatePayloads", "OutputTotalBytes", "OutputTotalPayloads", "OverflowPackets", "PacketLossPercent", "PATError", "PCRAccuracyError", "PCRError", "PIDError", "PMTError", "PTSError", "RecoveredPackets", "RoundTripTime", "SourceARQRecovered", "SourceARQRequests", "SourceBitRate", "SourceCATError", "SourceConnected", "SourceContinuityCounter", "SourceCRCError", "SourceDisconnections", "SourceDroppedPackets", "SourceDroppedPayloads",  "SourceFECPackets", "SourceFECRecovered", "SourceLatePayloads", "SourceMergeActive", "SourceMergeLatency", "SourceMergeStatusWarnMismatch", "SourceMergeStatusWarnSolo", "SourceMissingPackets", "SourceNotRecoveredPackets", "SourceOverflowPackets", "SourcePacketLossPercent", "SourcePATError", "SourcePCRAccuracyError", "SourcePCRError", "SourcePIDError", "SourcePMTError", "SourcePTSError", "SourceRecoveredPackets", "SourceRoundTripTime", "SourceSelected", "SourceTotalBytes", "SourceTotalPayloads", "SourceTotalPackets", "SourceTransportError", "SourceTSByteError", "SourceTSSyncLoss", "TotalPackets", "TransportError", "TSByteError", "TSSyncLoss", "Uptime"},
```

Comment (Deepali1211) [reply pkg/tsdb/cloudwatch/constants/metrics.go:360]: I fixed the alphabetization also.

Comment (Deepali1211) [reply pkg/tsdb/cloudwatch/constants/metrics.go]: okay done, sorted the list.

**Review by iwysiu — APPROVED** [review]

**Pull request merged**